test/integration_test/test_common.go

```diff
@@ -51,6 +56,62 @@ const (
 	defaultValidateUninstallRetryInterval = 30 * time.Second
 )
 
+// node* is to be used in the Node section of the StorageCluster spec. node0 will select the
```

Comment:
NB: renamed this file from `common_test` to `test_common` so that non-test files in the test folder can also use methods from here.